pkg/lint/rules/chartfile.go

```diff
@@ -104,9 +105,14 @@ func validateChartYamlFormat(chartFileError error) error {
 }
 
 func validateChartName(cf *chart.Metadata) error {
+	var validNameRegex = regexp.MustCompile(`^[a-z0-9](([-a-z0-9]*)?)$`)
+
 	if cf.Name == "" {
 		return errors.New("name is required")
 	}
+	if !validNameRegex.MatchString(cf.Name) {
+		return fmt.Errorf("invalid release name, must match regex %s", validNameRegex.String())
```

Comment:
Did you mean "invalid chart name" here?

```suggestion
		return fmt.Errorf("invalid chart name, must match regex %s", validNameRegex.String())
```